RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -145,14 +146,24 @@ struct SubscriptionDetailsView: View {
 
     let iconWidth = 22.0
     let subscriptionInformation: SubscriptionInformation
+    let localization: CustomerCenterConfigData.Localization
     let refundRequestStatusMessage: String?
 
     var body: some View {
         VStack(alignment: .leading, spacing: 8) {
             VStack(alignment: .leading) {
                 Text("\(subscriptionInformation.title)")
                     .font(.headline)
-                Text("\(subscriptionInformation.explanation)")
+
+                var explanation: String {
+                    return subscriptionInformation.active ? (
+                        subscriptionInformation.willRenew ?
+                        localization.commonLocalizedString(for: .subEarliestRenewal) :
+                            localization.commonLocalizedString(for: .subEarliestExpiration)
+                    ) : localization.commonLocalizedString(for: .subExpired)
+                }
```

Comment:
NABD but any reason to keep this as a computed property? Or could we simplify it like this
```suggestion
                var explanation = subscriptionInformation.active ? (
                     subscriptionInformation.willRenew ?
                        localization.commonLocalizedString(for: .subEarliestRenewal) :
                            localization.commonLocalizedString(for: .subEarliestExpiration)
                    ) : localization.commonLocalizedString(for: .subExpired)
```

Reply:
No reason! I even asked ChatGPT if this was idiomatic SwiftUI haha. Thanks for the suggestion! Please keep em coming.

I'll do this here and for `expirationString` below.

Reply:
This is now done: https://github.com/RevenueCat/purchases-ios/pull/4083/commits/2fc4bb667ed0daf5a80f0cab0b72161ed0999b6c